apps/src/templates/studioHomepages/TeacherHomepage.jsx

```diff
@@ -140,21 +133,8 @@ export default class TeacherHomepage extends Component {
         <ProtectedStatefulDiv
           ref="termsReminder"
         />
-
```

Comment:
I'm going to do a follow up PR that cleans up our announcement system on this page (takes out  DCDO census and hoc_launch conditionals and sets up a clean way to show the small announcement or the big announcement). To keep this change un-blocked I'm just completely removing the small Notification for now.